server/models/AssessmentDayDataModel/index.js

```diff
@@ -0,0 +1,15 @@
+const AssessmentDayDataModel = {
+  all: async (db, assessmentCollection) =>
+    await db
+      .collection(assessmentCollection)
+      .find({}, { projection: { _id: 0, path: 0 } })
+      .toArray(),
+  save: async (db, assessmentCollection, assessmentDayData) =>
```

Comment:
what do you think about a name like `saveMany` to denote that this method deals with more than 1 record?

Reply:
[c1cf0d0](https://github.com/AMP-SCZ/dpdash/pull/382/commits/c1cf0d04b9d9dd43b6aceef72cc4f1f45af67a9c)